diff --git a/packages/std/Cargo.toml b/packages/std/Cargo.toml
index c38e3933c..01b2e9d06 100644
--- a/packages/std/Cargo.toml
+++ b/packages/std/Cargo.toml
@@ -55,6 +55,7 @@ cosmwasm_2_1 = ["cosmwasm_2_0"]
 # This enables functionality that is only available on 2.2 chains.
 # It adds `IbcMsg::PayPacketFee` and `IbcMsg::PayPacketFeeAsync`.
 cosmwasm_2_2 = ["cosmwasm_2_1"]
+cosmwasm_2_3 = ["cosmwasm_2_2", "dep:cosmwasm-crypto-bn254"]
 # This enables functionality that is only available on 3.0 chains.
 # It adds `WasmQuery::RawRange`.
 cosmwasm_3_0 = ["cosmwasm_2_2"]
@@ -78,6 +79,7 @@ rmp-serde = "1.3.0"
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 bech32 = "0.11.0"
 cosmwasm-crypto = { workspace = true }
+cosmwasm-crypto-bn254 = { version = "0.1.0", path = "../crypto-bn254", optional = true }
 rand_core = { version = "0.6.4", features = ["getrandom"] }
 
 [dev-dependencies]
